daemon/MenuDaemon.vala

```diff
@@ -258,5 +268,65 @@ namespace Gala {
                 push_in = true;
             }, 3, Gdk.CURRENT_TIME);
         }
+
+        private void init_desktop_menu () {
```

Comment:
Since this is only ever done once, it probably doesn't need to be a separate method

Reply:
Solved!